freqtrade/optimize/backtesting.py

```diff
@@ -81,26 +78,20 @@ def get_trade_entry(pair, row, ticker, trade_count_lock, args):
                   )
 
     # calculate win/lose forwards from buy point
-    sell_subset = ticker[row.Index + 1:][['close', 'date', 'sell']]
+    sell_subset = ticker[ticker.date > row.date][['close', 'date', 'sell']]
     for row2 in sell_subset.itertuples(index=True):
         if max_open_trades > 0:
             # Increase trade_count_lock for every iteration
             trade_count_lock[row2.date] = trade_count_lock.get(row2.date, 0) + 1
 
-        current_profit_percent = trade.calc_profit_percent(rate=row2.close)
-        if (sell_profit_only and current_profit_percent < 0):
-            continue
-        if min_roi_reached(trade, row2.close, row2.date) or \
-            (row2.sell == 1 and use_sell_signal) or \
-                current_profit_percent <= stoploss:
-            current_profit_btc = trade.calc_profit(rate=row2.close)
+        buy_signal = rows[rows.date == row2.date].empty
+        if(should_sell(trade, row2.close, row2.date, buy_signal, row2.sell)):
             return row2, (pair,
-                          current_profit_percent,
-                          current_profit_btc,
-                          row2.Index - row.Index,
-                          current_profit_btc > 0,
-                          current_profit_btc < 0
-                          )
+                          trade.calc_profit_percent(rate=row2.close),
+                          trade.calc_profit(rate=row2.close),
+                          row2.Index - row.Index
+                          ), row2.date
+    return False
```

Comment:
remove return False ?
Either we return a tuple or None, I guess it's more type-cleanier.

Reply:
OK for me!